docs/contributing/contributing.md

```diff
@@ -45,19 +55,12 @@ DEFAULT_LOG_LEVEL_TEXT: ERROR
 ```
 
 * Never hardcode any file paths. Always use import variables such as `INPUT_DIR`, `OUTPUT_DIR` and use `os.path.join()` to make file paths. This ensures that the tests will run independent of the OS they are running on.
+* Running the test suite changes many files in `..\output\..` folders which messes up `git status`. 
```

Comment:
Could you add a note about why it is done like that? Could the output not be instead written to a temporary dir and then compared to the expected result in "output"? Alternatively, the output could be written to a git-ignored folder. (related #880)

Reply:
Issue 880 is a big priority, @hsolbrig is working on this. I think it may detract from this document to include too much historic context especially when we get rid of this.

For now, we can just add a comment this is a temporary situation

Reply:
Yup, it's probably not the best idea to add a lot of historic context to the document, only to deprecate it soon after.

I've added a comment to address your concern @dalito.